core/persistence-jpa/src/main/java/org/apache/syncope/core/persistence/jpa/entity/JPAAccessToken.java

```diff
@@ -44,7 +44,7 @@
     @Temporal(TemporalType.TIMESTAMP)
     private Date expiryTime;
 
-    @Column(nullable = true)
+    @Column(unique = true)
```

Comment:
As said during the initial mail exchange, I am not sure of the reason why owner is currently set as nullable, and I don't see enough reasons to change it, please revert.

Reply:
I think in JPA it is redundant to use nullable = true this is the default scenario or all columns in JPA. Should I still change it?

Reply:
Just revert, I know `nullable = true` is the default, but I don't see reasons to change.